lib/intent-models/asyncapi.js

```diff
@@ -0,0 +1,187 @@
+
+module.exports = {};  
+const IntentBase = require('./base');
+const { IntentChannel } = require('./channel');
+const { IntentMessage } = require('./message');
+const { SecurityScheme } = require('./security-scheme');
+const { IntentInfo } = require('./info');
+const { IntentOperation } = require('./operation');
+const Schema = require('../models/schema');
+const { IntentServer } = require('./server');
+const { IntentExternalDocument } = require('./external-document');
+const { IntentTag } = require('./tag');
+/**
+ * 
+ * @class
+ * @alias module:@asyncapi/parser#IntentAsyncAPIDocument
+ * @extends IntentBase
+ */
+class IntentAsyncAPIDocument extends IntentBase {
+  /**
+   * @returns {string}
+   */
+  version() {
+    return '2.0.0';
+  }
```

Comment:
Does it make sense to call it `version()` rather than what the spec uses `asyncapi()`?

Reply:
Also I think we can make this enum right?

Reply:
What's the benefit on doing enum? In fact it penalizes readability: `Version200` vs `'2.0.0` ?

Reply:
While you may read it that way, from tooling perspective it just becomes easier to check a statement such as `if(asyncapi.version() === AsyncAPI.versions.Version200)`
vs
`if(asyncapi.version() === "2.0.0")` where the developer need to look up what values this might be.

Reply:
@smoya do you disagree or? 😆

Reply:
LOL yeah I disagree. I'm leaning towards readability here.

Reply:
Can you explain why you think a string is more readable from the tooling perspective? 🤔

Reply:
I mean, tooling is written by people. People will prefer to read the version as plain string rather than a constant?
I mean, `AsyncAPI.versions.Version200` could mean version `200`? It is so hard to read.

Reply:
True, fair, lets leave it for now.